modcc/parser.cpp

```diff
@@ -399,9 +399,13 @@ void Parser::parse_state_block() {
         }
 
         parm.token = token_;
-        //state_block.state_variables.push_back(token_.spelling);
         get_token();
 
+        if(token_.type == tok::from) {
```

Comment:
Should this check for an incorrectly formed `FROM <value> To <value>` statement?
If an error was set, it should return, I think.

Reply:
This part is right, and you should call `from_to_description()` should call `error` if it detects an error in the statement.
You can then check if an error was set, and just `return`.